pybamm/solvers/algebraic_solver.py

```diff
@@ -24,14 +25,18 @@ class AlgebraicSolver(pybamm.BaseSolver):
         specified in the form "lsq_methodname"
     tol : float, optional
         The tolerance for the solver (default is 1e-6).
+    max_step : float, optional
```

Comment:
the algebraic solver doesn't step in time so it doesn't make sense for it to be an argument here